dist/angucomplete-alt.js

```diff
@@ -88,7 +94,7 @@ angular.module('angucomplete-alt').directive('angucompleteAlt', ['$q', '$parse',
           document.body.addEventListener('click', clickoutHandlerForDropdown);
         }
       } else {
-        config.mousedownOn = target.className;
+        config.mousedownOn = target.getAttribute('class');
```

Comment:
minor, this could just do `config.mousedownOn = className` since `className` is defined on line 62.